buyer_data/admin.py

```
@@ -1,7 +1,50 @@
 from django.contrib import admin
+from django.utils.html import format_html
 from .models import *
```

Flake8 (7.2.0)

[error] 'from .models import *' used; unable to detect undefined names

(F403)